tests/core/pyspec/eth2spec/test/utils/randomized_block_tests.py

```diff
@@ -195,6 +204,11 @@ def random_block_bellatrix(spec, state, signed_blocks, scenario_state):
     return block
 
 
+def random_block_capella(spec, state, signed_blocks, scenario_state):
+    block = random_block_bellatrix(spec, state, signed_blocks, scenario_state)
```

Comment:
`build_random_block_from_state_for_next_block()` (which is used under the hood) does not add `BLSToExecutionChange` operations. So we should conditionally add in that function or just add some random ones here (I think doing here is cleaner but then the original functoin name is slight off, it should maybe be `build_random_phase0_block..`

Reply:
added in https://github.com/ethereum/consensus-specs/pull/3040/commits/0e2c3d89e020f43e0c2af1c246ec772ec54ee250